circle.yml

```diff
@@ -17,9 +17,9 @@ test:
     #run tests
     - >
       if [[ $CIRCLE_BRANCH != *"master"* || !$CIRCLE_TAG ]] ; then
-        FEAT_TEST=true npm run test:e2e
+        E2EPROFILE=saucelight npm run test:e2e
       else
-        npm run test:e2e
+        E2EPROFILE=sauceextended npm run test:e2e
```

Comment:
i think we should set the E2EPROFILE for circleci, so you can define if you want to run differently from the default scenario. For defining environment variables you don't necessarily need to use circleci's UI, you can just do like this: https://github.com/purposeindustries/blog.intellyo.com/blob/master/circle.yml#L4-L19
(also i would encourage you doing that, so it would end up in git. The ui more like for setting secrets and such.)